### PR TITLE
caption update

### PR DESCRIPTION
**Pull request by pstq**
aaaaaaa

### CODE REVIEW

File: _slides/12-pstq.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I wanna be a GIT Ninja!
```
Comment (a-lai2) [inline _slides/12-pstq.md:6]: looks great

**Comment by randyb77** [comment]
Nice job Git Ninja.

**Review by a-lai2 — APPROVED** [review]
Approve

**Review by randyb77 — APPROVED** [review]
Okay, then.

File: _slides/12-pstq.md (hunk 1 above)
Comment (marcinhk) [inline _slides/12-pstq.md:6]: Love it!

**Pull request merged**